src/jira/client/jira-client.ts

```diff
@@ -475,6 +479,14 @@ export const getTruncatedIssuekeys = (data: IssueKeyObject[] = []): IssueKeyObje
 				issueKeys: value.issueKeys.slice(ISSUE_KEY_API_LIMIT)
 			});
 		}
+		const association = findIssueKeyAssociation(value);
+		if (association && association.values.length  > ISSUE_KEY_API_LIMIT) {
```

Comment:
Could the `association.values` be undefined?
e.g. `association.values?.length` instead

Reply:
good point